lib/lotus/cli.rb

```diff
@@ -72,14 +72,16 @@ def console
       `lotus new` creates a new lotus application.
       You can specify various options such as the database to be used as well as the path and architecture.
 
-      $ > lotus new fancy_app --application=container
+      $ > lotus new fancy_app --application=api
+
+      $ > lotus new fancy_app --arch=app
 
       $ > lotus new fancy_app --lotus_head
     EOS
 
-    method_option :database, desc: "application database (#{Lotus::Generators::DatabaseConfig::SUPPORTED_ENGINES.keys.join('/')})", default: 'filesystem'
-    method_option :architecture, desc: 'application architecture (container/app)', default: Lotus::Commands::New::Abstract::DEFAULT_ARCHITECTURE
-    method_option :application_name, desc: 'application name, only for container', default: Lotus::Commands::New::Container::DEFAULT_APPLICATION_NAME
+    method_option :database, aliases: ['-d', '--db'], desc: "application database (#{Lotus::Generators::DatabaseConfig::SUPPORTED_ENGINES.keys.join('/')})", default: 'filesystem'
+    method_option :architecture, aliases: ['-a', '--arch'], desc: 'application architecture (container/app)', default: Lotus::Commands::New::Abstract::DEFAULT_ARCHITECTURE
+    method_option :application_name, aliases: '--application', desc: 'application name, only for container', default: Lotus::Commands::New::Container::DEFAULT_APPLICATION_NAME
```

Comment:
IMHO, `--application` can be ambiguous because it might be mistaken with `architecture` option. I think it should be leaving as in or with alias `--app_name`